content/post/054-go-ipfs-0.4.22.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2019-08-06
+url: 054-go-ipfs-0.4.22
+title: go-ipfs 0.4.22 released
+author: Steven Allen
+---
+
+We're releasing a PATCH release of go-ipfs based on 0.4.21 containing some critical fixes.
+
+The IPFS network has scaled to the point where small changes can have a wide-reaching impact on the entire network. To keep this situation from escalating, we've put a hold on releasing new features until we can improve our [release process](https://github.com/ipfs/go-ipfs/pull/6482) (which we will be trialing in this release) and [testing procedures](https://github.com/ipfs/go-ipfs/issues/6483).
+
+This release includes fixes for the following regressions:
+
+1. A major bitswap throughput regression introduced in 0.4.21 ([ipfs/go-ipfs#6442](https://github.com/ipfs/go-ipfs/issues/6442)).
+2. High bitswap CPU usage when connected to many (e.g., 10,000) peers. See [ipfs/go-bitswap#154](https://github.com/ipfs/go-bitswap/issues/154).
+2. The local network discovery service sometimes initializing before the networking module, causing it to announce the wrong addresses and sometimes complain about not being able to determine the IP address ([ipfs/go-ipfs#6415](https://github.com/ipfs/go-ipfs/pull/6415)).
```

Comment:
```suggestion
2. The local network discovery service sometimes initializes before the networking module, causing it to announce the wrong addresses and sometimes complain about not being able to determine the IP address ([ipfs/go-ipfs#6415](https://github.com/ipfs/go-ipfs/pull/6415)).
```

This reads ok as it is assuming you've read "This release includes fixes for the following regressions:" beforehand (you should have, but you might be skimming), if not then "The local network discovery service sometimes initializing" sounds incorrect. This way it makes sense both ways.